manifests/o/OpenVPNTechnologies/OpenVPN/2.5.037/OpenVPNTechnologies.OpenVPN.locale.en-US.yaml

```
@@ -0,0 +1,69 @@
+# Created using wingetcreate 1.1.2.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.2.0.schema.json
+
+PackageIdentifier: OpenVPNTechnologies.OpenVPN
+PackageVersion: 2.5.037
+PackageLocale: en-US
+Publisher: OpenVPN, Inc.
+PublisherUrl: https://openvpn.net
+PublisherSupportUrl: https://forums.openvpn.net
+PrivacyUrl: https://openvpn.net/privacy-policy/
+Author: Open VPN Technologies
+PackageName: OpenVPN
+PackageUrl: https://openvpn.net
+License: GNU GPLv2
+LicenseUrl: https://github.com/OpenVPN/openvpn/blob/master/COPYING
+Copyright: Copyright © 2021 OpenVPN Inc
+CopyrightUrl: https://openvpn.net/license/
+ShortDescription: Create secure VPN connections over the internet with this VPN-software
+Description: OpenVPN provides flexible VPN solutions to secure your data communications, whether its for Internet privacy, remote access for employees, securing IoT, or for networking Cloud data centers.
+Moniker: openvpn
+Tags:
+- community
+- connection
+- foss
+- network
+- open-source
+- privacy
+- private
+- private-connection
+- private-network
+- security
+- ssl
+- tunnel
+- virtual-private-connection
+- vpn
+ReleaseNotes: |

```

```suggestion
ReleaseNotes: |-
```
Nit: Best to add the `-` for consistency